sentry_sdk/tracing_utils.py

```diff
@@ -475,15 +475,14 @@ def serialize(self, include_third_party=False):
         return ",".join(items)
 
 
-def should_propagate_trace(hub, url):
-    # type: (sentry_sdk.Hub, str) -> bool
+def should_propagate_trace(client, url):
+    # type: (sentry_sdk.client.BaseClient, str) -> bool
```

Comment:
We should either ensure this method remains compatible with the old Hub API, or make sure we call out in the migration guide that from 2.0, it only accepts client objects, not hubs.

Reply:
As discussed, we will mention it in the migration guide!

Reply:
Should we add this into the migration guide already in this PR?